BrainPortal/db/migrate/20191025141308_add_position_to_users.rb

```diff
@@ -0,0 +1,20 @@
+class AddPositionToUsers < ActiveRecord::Migration[5.0]
+
+  def up
+    add_column :users, :position, :string, :after => :full_name
+
+    User.all.each do |user|
```

Comment:
I'm not getting any `position` information transfered to the User records, not sure why. All I get is 'Unknown' for everything.